rados/rados_getaddrs.go

```diff
@@ -0,0 +1,31 @@
+//go:build ceph_preview
+// +build ceph_preview
```

Comment:
I think we can safely ignore adhering to the backward compatible variant these days.